GeneratorInterface/Hydjet2Interface/src/Hydjet2Hadronizer.cc

```diff
@@ -408,7 +416,10 @@ bool Hydjet2Hadronizer::get_particles(HepMC::GenEvent *evt) {
       isub_l = hj2->GetiJet().at(ihy);
     }
 
-    if ((hj2->GetFinal().at(ihy)) == 1)  //convertStatus(hj2->GetPythiaStatus().at(ihy)) == 1)
+    //if ((hj2->GetType().at(ihy)) == 1)//hydro (hj2->GetFinal()): 1 - stable, 0 - decayed
```

Comment:
Is this commented out code needed? Please add another line of comment which explains why it is needed to be kept here, otherwise just remove it